pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
```

Comment:
@thomastaylor312 In fact, the reason I make this PR was to work towards a standard, which other charts can follow, so I'm hoping we can create a good mvp here that covers the common use cases 😄